sys/parsoid_proxy.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+/*
+ * Simple proxy to route requests to the client-requested
+ * Parsoid variant (JS or PHP) during the transistion
```

Comment:
`s/transistion/transition/`

Reply:
Oh, boy! VS Code does not come with a spell checker. Adding one. :)